assets/js/util.js

```diff
@@ -308,17 +332,21 @@ function callApy(options, endpoint) {
     var requestUrl = window.location.protocol + window.location.hostname +
         window.location.pathname + '?' + $.param(requestOptions.data);
 
+    if(useAjax) {
+        return $.ajax(requestOptions);
```

Comment:
Hmm... this doesn't preserve the behavior where we use POST instead of GET, right?